docs/how_to_guides/how_to_process_previously_recorded_videos.md

```diff
@@ -1,54 +1,82 @@
 
 # How to process pre-recorded synchronized videos with `alpha` GUI
 
-Here's a workflow you can use to pre-process videos recorded with the `pre-alpha (<=v0.0.54)` using the `alpha` 
+This is a workflow you can use to process **any** pre-recorded & synchronized videos.
+
+!!! take-note "Note"
+
+    This software is a work-in-progress, you'll likely have to click around and test things out to get this process to work for the first time. Often, you might click a button and want to go back, but the GUI doesn't have "go-back" capabilities yet :) So in order to go back, simply restart the GUI. You can quickly restart the GUI at any time by selecting the GUI and pressing `ctrl-R`
+
+??? pre-alpha "pre-alpha tips"
+
+    If you've pre-recorded videos using freemocap `pre-alpha (<=v0.0.54)`, these "pre-alpha tips" boxes will contain info to specifically address a "`pre-alpha` -> `alpha GUI`" workflow!
 
 ## Step 1 - Run the GUI  
-Instructions here - https://github.com/freemocap/freemocap#how-to-run-the-alpha-gui 
+Detailed instructions on how to run the GUI using anaconda can be found [on our GitHub](https://github.com/freemocap/freemocap#how-to-run-the-alpha-gui). 
 
-- run  `python src/gui/main/main.py` in a terminal with the proper environment activated
--  a GUI should pop up that looks like this: 
+- Once you have the GUI set up and ready to go, run  `python src/gui/main/main.py` in a terminal with the proper environment activated.
+
+!!! take-note "Note"
+    
+    If the GUI fails to run, your version of freemocap might be out of date. You can `git fetch` to make sure your repository is up to date, or you can re-clone the freemocap `main` repository.
+-  A GUI should pop up that looks like this: 
   
 ![image](https://user-images.githubusercontent.com/15314521/201449304-4a26d703-e971-404f-81f4-a70d042f9e66.png)
```

Comment:
Gonna want to grab some new screenshots showing the latest version of the GUI (i.e. I removed the `4-Visualized..` tab) 

*maybe* (if it's not too cluttered looking) a call out saying the date the screenshot was grabbed and a note saying that it might look slightly different if they're reading that tutorial noticeably after that date 

(we should endeavor to keep these screenshots as up-to-date as possible)